src/validation/models/stopValidatorModel.ts

```diff
@@ -0,0 +1,10 @@
+const nameRule = 'required|min:1|max:20|string'; // TODO: These are just made up
```

Comment:
Why was this file renamed?

Reply:
Because it really is stop validationModel, all validations are for stop properties

Reply:
you renamed the file from stopValidationModel to stopValidatorModel. All other models have validation not validator.

Reply:
Fixed now, my bad